src/hotspot/share/opto/vectornode.hpp

```diff
@@ -910,6 +911,10 @@ class LoadVectorGatherNode : public LoadVectorNode {
             ((is_subword_type(vect_type()->element_basic_type())) &&
               idx == MemNode::ValueIn + 1);
   }
+  virtual int store_Opcode() const {
+    // Ensure it is different from any store opcode
+    return Op_LoadVectorGather;
```

Comment:
OK. I was just a bit concerned that a check for equality between 2 `store_Opcode` could be true because they are both -1 but this shouldn’t happen. Changed.